src/tls_wrap.cc

```diff
@@ -930,6 +930,16 @@ void TLSWrap::EnableKeylogCallback(
 # define HAVE_SSL_TRACE 0
 #else
 # define HAVE_SSL_TRACE 1
+static void TraceMsgCallback(
```

Comment:
Why not use the downstream names?
```c++
void SSL_trace(int write_p, int version, int content_type,
               const void *buf, size_t len, SSL *ssl, void *arg);
```
<sub>we write code once and read it dozens of times</sub>

Reply:
P.S. if it is just a one-and-done callback, maybe make it a lambda.

Reply:
I'll try a lambda, I don't know how they work with external C libs, or multiple statements. Personally, I think spelling out a bunch of names that are irrelevant to the code here forces people to read code they don't need to concern themselves with, so its a read-optimization, not write optimization. This would be an ideal place to use `...`, but that's not supported yet in C++, to my knowlege.

Reply:
ok, it has verbose names and a lambda

Reply:
@refack PTAL

Reply:
```
/home/travis/build/nodejs/node/out/Release/.deps//home/travis/build/nodejs/node/out/Release/obj.target/libnode/src/node_report.o.d.raw   -c
../src/tls_wrap.cc: In lambda function:
../src/tls_wrap.cc:944:75: error: expected ‘{’ before ‘void’
           content_type, const void* buf, size_t len, SSL* ssl, void* arg) void {
                                                                           ^~~~
../src/tls_wrap.cc: In static member function ‘static void node::TLSWrap::EnableTrace(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/tls_wrap.cc:944:75: error: expected ‘)’ before ‘void’
```

gcc-8 had not trouble with this, but it failed to compile on Travis. I'm trying a different syntax, it looks like depending on recent C++ language features might be unstable.

Reply:
Latest imp LGTM, and seems to be green on travis https://travis-ci.com/nodejs/node/builds/113349340